go.mod

```diff
@@ -8,21 +8,23 @@ require (
 	github.com/JeffreyRichter/enum v0.0.0-20180725232043-2567042f9cda
 	github.com/cpuguy83/go-md2man v1.0.10 // indirect
 	github.com/danieljoos/wincred v1.0.1
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
>github.com/davecgh/go-spew v1.1.1 // [](start = 1, length = 36)

the third party notice file needs to be updated accordingly

Reply:
Removed go-spew as it wasn't necessary. It's a leftover dep of minio